vyper/contracts/libs/ConsiderationBase.vy

```diff
@@ -0,0 +1,203 @@
+# @version 0.3.3
+
+from ..interfaces import ConduitControllerInterface
+
+MAX_BYTE_LENGTH: constant(uint8) = 100
```

Comment:
`offerItemTypeString` has a length of 106, `considerationItemTypeString` has a length of 132, and `orderComponentsPartialTypeString` has a length of 210